jedi/api/environment.py

```diff
@@ -363,18 +366,18 @@ def get_system_environment(version):
     raise InvalidPythonEnvironment("Cannot find executable python%s." % version)
 
 
-def create_environment(path, safe=True):
+def create_environment(path, env_vars={}, safe=True):
```

Comment:
One more small thing: can you create it like this (the current implementation would break the API BTW:

```
def create_environment(path, safe=True, **kwargs):
    return _create_environment(path, safe, **kwargs)  # TODO Python 2...


def _create_environment(path, safe=True, env_vars={}):
```

I want to make env_vars a keyword argument when I drop Python 2 (very soon).

Reply:
Done. Thanks for the feedback.